plugins/a11y-keyboard/msd-a11y-keyboard-atspi.c

```diff
@@ -0,0 +1,190 @@
+/* -*- Mode: C; tab-width: 8; indent-tabs-mode: nil; c-basic-offset: 8 -*-
+ *
+ * Copyright (C) 2020 Colomban Wendling <cwendling@hypra.fr>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2, or (at your option)
+ * any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA.
+ */
+
+#include "config.h"
+
+#include "msd-a11y-keyboard-atspi.h"
+
+#include <glib-object.h>
+
+#if HAVE_LIBATSPI
+#include <gdk/gdk.h>
+#include <atspi/atspi.h>
+
+/* See https://gitlab.gnome.org/GNOME/at-spi2-core/-/issues/22 */
+#define DESTROYING_ATSPI_LISTENER_CRASHES \
+        (! (defined (HAVE_LIBATSPI_NOBUG22) && HAVE_LIBATSPI_NOBUG22))
```

Comment:
This is sad, but unfortunately there's no released version with a fix, so we'll have to live with the workaround.